stellargraph/mapper/padded_graph_generator.py

```diff
@@ -91,8 +95,8 @@ def flow(
         with the supplied graph indexes and targets.
 
         Args:
-            graph_ilocs (iterable): an iterable of graph indexes in self.graphs for the graphs of interest
-                (e.g., training, validation, or test set nodes).
+            graph_ilocs (iterable): an iterable of graph indexes in self.graphs or an iterable of `StellarGraph`s
```

Comment:
Tiny thing, but we should consider renaming this parameter to `graphs`, so the error messages from `self._check_graphs` match.

```suggestion
            graphs (iterable): an iterable of graph indexes in self.graphs or an iterable of `StellarGraph`s
```

and all the other instances of `graph_ilocs` in this file.

Reply:
This breaks code specifying `graph_ilcs` as a kwarg but `PaddedGraphGenerator` wasn't in the last release so I think this should be fine.